viewer_controls.md

```diff
@@ -0,0 +1,296 @@
+  - updates value in text fields on right of the slider
+- Action
+ - enable Preview, Accept, Revert buttons
+
+**Preview button**
+- Disabled
+ - default
+- Enabled
+ - when the settings have been changed
+- Action
+ - trigger a "render" image event
+ - trigger a modification of the enabled flag of the Save button
+ - trigger a modification of the enabled flag of the Undo button
+
+**Accept button**
+- Disabled
+ - default
+- Enabled
+ - when the settings have been changed
+- Action
+ - trigger a "render" image event
+ - trigger a modification of the enabled flag of the Save button
+ - trigger a modification of the enabled flag of the Undo button
+ - background of the button and the slider updated
+ - close color picker dialog
+
+**Cancel button**
+- Enabled
+ - all the time
+- Action
+ - revert to rendering settings at time of opening of color picker 
+ - update color picker display
+ - trigger rendering settings update - ? only if Preview had been used
+
+#### Controls for all channels
+
+**Greyscale checkbox**
+- Unchecked
+ - default
+- Event - check
+ - toggles all on channels off
+- set rendering to grey scale
+- Event - uncheck
+ - toggles all channels that were on back on
+ - set rendering to previous
+- Action
+ - trigger a "render" image event
+ - trigger a modification of the enabled flag of the Save button
+ - trigger a modification of the enabled flag of the Undo button
+
+For all:
+
+- Enabled
+ - all the time
+
+**Min/Max button**
+- Action
+ - Set the start/end values of the intensity interval to the global min/max
+ - trigger a "render" image event
+ - trigger a modification of the enabled flag of the Save button
+ - trigger a modification of the enabled flag of the Undo button
+- Note
+ - global min/max are calculated at import if ``calculate stats`` is on. The default values are 0 and 1.
+
+**Full Range button**
+- Action
+ - Set the start/end values of the intensity interval to the min/max values of the pixels interval e.g. 0 to 65535 for 16 bit pixels type
+ - trigger a "render" image event
+ - trigger a modification of the enabled flag of the Save button
+ - trigger a modification of the enabled flag of the Undo button
+
+**Imported button**
+- Action
+ - will request new rendering settings - this means that z/t, color, pixels intensity values will be changed, the mapping family used for each channel - all settings need to be re-initialized.
+ - trigger a "render" image event
+ - trigger a modification of the enabled flag of the Save button
+ - trigger a modification of the enabled flag of the Undo button
+
+### Users settings
+
+**User setting thumbnail and text**
+- thumbnail displayed is generated using the last settings saved by the users who viewed the image
+- current implementation
+ - one thumbnail per user
+- text displayed under the thumbnail
+ - from string composed of the first name and last name of the user who viewed the image and saved settings
+- thumbnail selected
+ - all settings re-initialized
```

Comment:
we add a blue border around the clicked thumbnail that remains there until user saves settings - then blue border reverts to the user's own thumbnail.